core/app/class-orbit-fox-admin.php

```diff
@@ -454,12 +454,8 @@ public function get_recommended_plugins() {
 		foreach ( $th_plugins as $plugin_slug => $plugin_data ) {
 			$data[ $plugin_slug ] = $plugin_data;
 		}
-		uksort(
-			$data,
-			function() {
-				return rand() <=> rand();
-			}
-		);
+
+		shuffle( $data );
 
 		return $data;
```

Comment:
@UVLabs we can return shuffle( $data ) directly.

Reply:
@cristian-ungureanu it returns a bool but we need the array

Reply:
I see, thank you for checking